Program/C_Interface.cpp

```diff
@@ -86,30 +86,31 @@ extern "C" Solution *solve_cvrp(
 		std::vector<double> service_time(serv_time, serv_time + n);
 		std::vector<double> demands(dem, dem + n);
 
-		std::vector < std::vector< double > > dist_mtx = std::vector < std::vector< double > >(n + 1, std::vector <double>(n + 1));
-		for (int i = 0; i <= n; i++)
+		std::vector<std::vector<double> > distance_matrix(n, std::vector<double>(n));
+		for (int i = 0; i < n; i++)
 		{
-			for (int j = 0; j <= n; j++)
+			for (int j = 0; j < n; j++)
```

Comment:
I did not understand very well this change to strict inequality, if "n" is the number of customers, then the loop usually goes from 0 (depot), then 1 to n, involving a smaller or equal? Is "n" now the total number of nodes in the graph (including depot + customers) ?

Reply:
Yes, here `n` is the total number of nodes. Previously I used `n+1` by mistake. Luckily it didn't create any mistake, because the additional unit simply wasn't used :)

Reply:
OK, good catch!